docker-compose.yml

```diff
@@ -69,12 +69,15 @@ services:
       - COMPANY_HOUSE_API_URL=https://api.company-information.service.gov.uk
       - DEMO_MODE=true
       - SMTP_SECURE=false
-      - LOCAL_REGISTRY_TO_USE=GB
-      - SOCRATA_API_URL=https://data.ny.gov/resource/p66s-i79p.json
+      - LOCAL_REGISTRY_TO_USE=company_house
+      - NY_STATE_API_URL=https://dev.socrata.com/foundry/data.ny.gov/p66s-i79p
       - COMPANY_PROFILE_API_KEY=${COMPANY_PROFILE_API_KEY}
       - IPID_API_URL=https://sandbox.ipid.works
       - IPID_API_KEY=${IPID_API_KEY}
       - IPID_CUSTOMER_ID=digicatapult-uat
+      - LOCAL_REGISTRY_COUNTRY_CODE=GB
+      - OPEN_CORPORATES_API_URL=http://wiremock:8080
+      - OPEN_CORPORATES_API_KEY=test-key
```

Comment:
could this be `OPEN_CORPORATES_API_KEY=${OPEN_CORPORATES_API_KEY}` so the env gets pulled in when local deving?